specreduce/tracing.py

```diff
@@ -55,6 +56,20 @@ def _bound_trace(self):
         ny = self.image.shape[0]
         self.trace = np.ma.masked_outside(self.trace, 0, ny-1)
 
+    def __add__(self, delta):
+        """
+        Return a copy of the trace shifted by delta pixels perpendicular to the axis being traced
+        """
+        copy = deepcopy(self)
+        copy.shift(delta)
+        return copy
+
+    def __sub__(self, delta):
+        """
+        Return a copy of the trace shifted by delta pixels perpendicular to the axis being traced
+        """
+        return self.__add__(-delta)
```

Comment:
It may be helpful if these docstrings noted the (opposite) directions on which these methods act, as well as the axis.